tests/generic_views/test_base.py

```diff
@@ -233,6 +233,32 @@ def test_args_kwargs_request_on_self(self):
             self.assertNotIn(attribute, dir(bare_view))
             self.assertIn(attribute, dir(view))
 
+    def test_overridden_setup(self):
+        class SetAttributeMixin:
+            def setup(self, request, *args, **kwargs):
+                self.attr = True
+                super().setup(request, *args, **kwargs)
+
+        class CheckSetupView(SetAttributeMixin, SimpleView):
+            def dispatch(self, request, *args, **kwargs):
+                assert hasattr(self, 'attr')
+                return super().dispatch(request, *args, **kwargs)
+
+        response = CheckSetupView.as_view()(self.rf.get('/'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_not_calling_parent_setup_error(self):
+        class TestView(View):
+            def setup(self, request, *args, **kwargs):
+                pass  # Not calling supre().setup()
```

Comment:
`s/supre/super/`

Reply:
(was fixed in b916c27f9a83c019ff85132712267508f126ca83)